runtime/binding-grpc/src/main/java/io/aklivity/zilla/runtime/binding/grpc/internal/config/GrpcBindingConfig.java

```diff
@@ -107,13 +127,18 @@ public GrpcMethodResult resolveMethod(
             final CharSequence serviceName = serviceNameHeader != null ? serviceNameHeader : matcher.group(SERVICE_NAME);
             final String methodName = matcher.group(METHOD);
 
-            final GrpcMethodConfig method = options.protobufs.stream()
-                .map(p -> p.services.stream().filter(s -> s.service.equals(serviceName)).findFirst().orElse(null))
-                .filter(Objects::nonNull)
-                .map(s -> s.methods.stream().filter(m -> m.method.equals(methodName)).findFirst().orElse(null))
-                .filter(Objects::nonNull)
-                .findFirst()
-                .orElse(null);
+            for (GrpcCatalogSchema catalog : catalogs)
+            {
+                catalog.resolveProtobuf();
+            }
+
+            final Stream<GrpcProtobufConfig> objectStream = catalogs.stream().map(c -> c.protobuf);
```

Comment:
Not sure about this. My idea is to reuse resolveMethod.

Reply:
If we use `catalogs` in `resolveMethod` then it can be

```
        return catalogs.stream().
            .map(c -> c.protobuf)
            .map(p -> p.services.stream().filter(s -> s.service.equals(serviceName)).findFirst().orElse(null))
            .filter(Objects::nonNull)
            .map(s -> s.methods.stream().filter(m -> m.method.equals(methodName)).findFirst().orElse(null))
            .filter(Objects::nonNull)
            .findFirst()
            .orElse(null);
```